packages/cli/src/active-executions.ts

```diff
@@ -94,13 +94,15 @@ export class ActiveExecutions {
 			await this.executionRepository.updateExistingExecution(executionId, execution);
 		}
 
+		const resumingExecution = this.activeExecutions[executionId];
```

Comment:
Should we use `ActiveExecutions.getExecution` every time we access on?

Reply:
`getExecution` throws, and in this case we don't want that.

Reply:
It'd be nice to rename to `getExecutionOrFail`